eslint-warning-thresholds.json

```diff
@@ -134,10 +134,10 @@
     "jsdoc/check-tag-names": 2
   },
   "packages/base-controller/src/Messenger.test.ts": {
-    "import-x/namespace": 33
+    "import-x/namespace": 34
```

Comment:
Any way we can fix these lint warnings? We don't want to allow these numbers to get bigger.